adapters/mqtt-vertx-base/src/main/java/org/eclipse/hono/adapter/mqtt/AbstractVertxBasedMqttProtocolAdapter.java

```diff
@@ -13,6 +13,10 @@
 
 package org.eclipse.hono.adapter.mqtt;
 
+import static io.netty.handler.codec.mqtt.MqttConnectReturnCode.CONNECTION_REFUSED_BAD_USER_NAME_OR_PASSWORD;
```

Comment:
we do not statically import constants, because we find it important to see where the constant comes from in the code.

Reply:
Fixed.